src/form/editors/NumberEditorView.js

```diff
@@ -48,7 +52,9 @@ export default (formRepository.editors.Number = BaseItemEditorView.extend({
     },
 
     initialize(options) {
-        if (options.format) {
+        this.format = options.format || options.options;
+        if (this.format) {
+            this.intl = new Intl.NumberFormat(Core.services.LocalizationService.langCode, options.options);

```

Comment:
options.options?

Reply:
![image](https://user-images.githubusercontent.com/34572116/43705608-a6d93bbe-996b-11e8-94f8-9c6390d79bf9.png)

Reply:
rename it!)